src/olympia/accounts/tests/test_helpers.py

```diff
@@ -0,0 +1,23 @@
+from django.test import RequestFactory
+
+import mock
+
+from olympia.accounts import helpers
+
+
+@mock.patch(
+    'olympia.accounts.helpers.utils.default_fxa_login_url',
+    lambda c: 'http://auth.ca')
+def test_login_link():
+    request = RequestFactory().get('/en-US/firefox/addons')
```

Comment:
If we're doing `auth.ca` wouldn't this likely be `en-GB`, `fr-CA`, or maybe even the neglected `en-CA`?

😉